### PR TITLE
Add 2 templates elementor.com

### PR DESCRIPTION
**Pull request by elementortzvi**
Add 2 templates
thanks

### CODE REVIEW

**Comment by yotamselementor** [comment]
@pawel-kow any chance you take a look at this?

File: elementor.com.website.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{  
+   "providerId":"elementor.com",
+   "providerName":"Elementor",
+   "serviceId":"website",
+   "serviceName":"Elementor Website",
+   "version": 1,
+   "logoUrl":"https://elementor.com/marketing/wp-content/uploads/2021/10/Elementor-Logo-Full-Red.png",
+   "description":"Enables a domain to work with Elementor",
+   "syncPubKeyDomain": "domainconnect.elementor.com",
+   "hostRequired":true,
```
Comment (pawel-kow) [inline elementor.com.website.json:10]: This template looks like an APEX template, so I think `hostRequired` = false is what you want.

File: elementor.com.cname.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+   "providerId":"elementor.com",
+   "providerName":"Elementor",
+   "serviceId":"cname",
+   "serviceName":"Elementor hosting",
+   "version": 1,
+   "logoUrl":"https://elementor.com/marketing/wp-content/uploads/2021/10/Elementor-Logo-Full-Red.png",
+   "description":"Enables a domain to work with Elementor",
+   "syncPubKeyDomain": "domainconnect.elementor.com",
+   "hostRequired":true,
+   "records":[
```
Comment (pawel-kow) [inline elementor.com.cname.json:11]: It's generally useful to add `variableDescription` if there are variables.

File: elementor.com.website.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{  
+   "providerId":"elementor.com",
+   "providerName":"Elementor",
+   "serviceId":"website",
+   "serviceName":"Elementor Website",
+   "version": 1,
+   "logoUrl":"https://elementor.com/marketing/wp-content/uploads/2021/10/Elementor-Logo-Full-Red.png",
+   "description":"Enables a domain to work with Elementor",
+   "syncPubKeyDomain": "domainconnect.elementor.com",
```
Comment (pawel-kow) [inline elementor.com.website.json:9]: It's generally useful to add `variableDescription` if there are variables.

**Review by pawel-kow — CHANGES_REQUESTED** [review]
Sorry for delay in the review. Vacation + sick leave.

**Review by pawel-kow — APPROVED** [review]
LGTM!

**Comment by yotamselementor** [comment]
> LGTM!

Thanks @pawel-kow , feel well!

**Pull request merged**